src/ui/osx/TogglDesktop/test2/NSTextField+Ext.m

```diff
@@ -0,0 +1,22 @@
+//
+//  NSTextField+Ext.m
+//  TogglDesktop
+//
+//  Created by Nghia Tran on 11/20/18.
+//  Copyright © 2018 Alari. All rights reserved.
+//
+
+#import "NSTextField+Ext.h"
+
+@implementation NSTextField(Extension)
+
+-(void)resetCursorColor
+{
+    NSTextView *textField = (NSTextView *)[self currentEditor];
+    if ([textField respondsToSelector:@selector(setInsertionPointColor:)])
+    {
+        [textField setInsertionPointColor:[self textColor]];
```

Comment:
We can't use system defined text color here as we do not currently support macOs Dark mode and it will we white on white if the user has Dark mode enabled.

You can use hex color with `[ConvertHexColor hexCodeToNSColor:color]];` method.

Reply:
If the app runs on OS Mojave, and using System Text Color, it means that the app is automatically adopted the Dark Mode. (See the screenshots). The `textColor` will change depend on the system Mode.
<img width="586" alt="screen shot 2018-11-20 at 16 24 20" src="https://user-images.githubusercontent.com/5878421/48764144-6fa9dd00-ece1-11e8-8203-c3f966cac796.png">
<img width="575" alt="screen shot 2018-11-20 at 16 24 24" src="https://user-images.githubusercontent.com/5878421/48764145-70427380-ece1-11e8-8531-42ee9e590fe4.png">

So, by setting the [self textColor], it guarantees that the Cursor Color is always same with Text Color, means it's visible all time regardless of white/dark mode.

I tested it in both mode, and it worked as our exceptions. Or I'm missing something? 🤔

Reply:
Although, if the app is built on Xcode 9.3, and run on OS Mojave, then the value of `[self textColor]` guarantees that the color of cursor color is same with the text color in Email. It means that the cursor is always visible.

As a result, it still corrects IMO